### ISSUE TITLE
Extracting Attention !

### ISSUE THREAD

**Issue by jaandoui** [MAGICS-LAB/DNABERT_2#82]
Hi! 
I'm trying to get the Attention matrices but they are not returned, I use output_attentions=True, but the only tuples returned are the the logits and the hidden_layers. 
How can we return the attention too ? 

I hope you can help me with that! 

Thank you!
